contentcuration/contentcuration/frontend/channelEdit/components/Clipboard/index.vue

```diff
@@ -278,12 +273,11 @@
         'resetPreloadClipboardNodes',
       ]),
       refresh() {
-        if (this.refreshing) {
+        if (this.initializing) {
           return;
         }
 
-        this.refreshing = true;
-        this.loadChannels().then(() => (this.refreshing = false));
+        this.initialize();
```

Comment:
Added clipboard state variable `initializing` and `initialize` action for clarity since now the action is only used for this purpose.